{"name": "用户注册/重复注册-预期成功", "status": "failed", "statusDetails": {"message": "AssertionError: 接口返回码是 【 2005 】, 返回信息：手机号已被注册！！！ \nassert False is True\n +  where False = <core.result_base.ResultBase object at 0x1040de7f0>.success", "trace": "self = <testcases.scenario_test.test_04_repeat_register.TestRepeatReg object at 0x104206610>\ntestcase_data = {'address': '深圳市宝安区', 'except_code': 2002, 'except_msg': '用户名已存在，注册失败', 'except_result': False, ...}\n\n    @allure.story(\"用例--注册/重复注册--预期成功\")\n    @allure.description(\"该用例是针对 注册-重复注册 场景的测试\")\n    @allure.issue(\"https://blog.csdn.net/\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://blog.csdn.net/\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"用户注册/重复注册-预期成功\")\n    @pytest.mark.single\n    @pytest.mark.usefixtures(\"delete_register_user\")\n    def test_user_repeat_register(self, testcase_data):\n        username = testcase_data[\"username\"]\n        password = testcase_data[\"password\"]\n        telephone = testcase_data[\"telephone\"]\n        sex = testcase_data[\"sex\"]\n        address = testcase_data[\"address\"]\n        except_result = testcase_data[\"except_result\"]\n        except_code = testcase_data[\"except_code\"]\n        except_msg = testcase_data[\"except_msg\"]\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = register_user(username, password, telephone, sex, address)\n        step_1(username, password, telephone, sex, address)\n>       assert result.success is True, result.error\nE       AssertionError: 接口返回码是 【 2005 】, 返回信息：手机号已被注册！！！ \nE       assert False is True\nE        +  where False = <core.result_base.ResultBase object at 0x1040de7f0>.success\n\ntestcases/scenario_test/test_04_repeat_register.py:41: AssertionError"}, "description": "该用例是针对 注册-重复注册 场景的测试", "steps": [{"name": "步骤1 ==>> 注册用户", "status": "passed", "parameters": [{"name": "username", "value": "'测试test'"}, {"name": "password", "value": "'123456'"}, {"name": "telephone", "value": "'13599999999'"}, {"name": "sex", "value": "'1'"}, {"name": "address", "value": "'深圳市宝安区'"}], "start": 1744708531878, "stop": 1744708531879}], "attachments": [{"name": "log", "source": "4a47e6f8-493d-4d41-9d9f-880e335e55ef-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "e8579c31-4b04-4d08-a2b3-50b572c945e9-attachment.txt", "type": "text/plain"}], "start": 1744708531793, "stop": 1744708531880, "uuid": "fe8d0b83-713c-4ba4-be00-7f114a8a1c19", "historyId": "01c293d6b5196fdce6a8cb6c26807156", "testCaseId": "2719552ca82c5b70e1485cdc5805d93f", "fullName": "testcases.scenario_test.test_04_repeat_register.TestRepeatReg#test_user_repeat_register", "labels": [{"name": "epic", "value": "针对业务场景的测试"}, {"name": "severity", "value": "critical"}, {"name": "story", "value": "用例--注册/重复注册--预期成功"}, {"name": "feature", "value": "场景：用户注册-重复注册"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.scenario_test"}, {"name": "suite", "value": "test_04_repeat_register"}, {"name": "subSuite", "value": "TestRepeatReg"}, {"name": "host", "value": "192.168.10.5"}, {"name": "thread", "value": "24590-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.scenario_test.test_04_repeat_register"}], "links": [{"type": "test_case", "url": "https://blog.csdn.net/", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://blog.csdn.net/", "name": "点击，跳转到对应BUG的链接地址"}]}